packages/cetlib/package.py

```diff
@@ -94,9 +94,11 @@ def setup_dependent_build_environment(self, env, dependent_spec):
         # Cleanup.
         sanitize_environments(env, "PERL5LIB")
 
-    def setup_dependent_run_environment(self, env, dependent_spec):
+    def setup_run_environment(self, env):
```

Comment:
- Already defined in line 83 above
- I don't believe `cetlib` provides any executables (might be wrong), so `PATH` should not be adjusted
- `cetlib` does not provide any plugins at runtime, so `CET_PLUGIN_PATH` should not be adjusted.